### PR TITLE
[W5][W13-3]Terh Jun Wei

### PR DESCRIPTION
**Pull request by fterh**
Add `count` command to quickly check the size of the address book.

### CODE REVIEW

**Comment by case141** [comment]
Good command to add for addressbook. :)

Suggestions: Since it is a simple command, it can be **combined** with the list command such that user can know the total number of records they are listing at the same time, for example:
[Command entered:  list]
You have successfully pulled a total of 56 records! 
 	1. Adam Brown Phone: 111111 Email: adam@gmail.com Address: 111, alpha street Tags: 
 	2. Betsy Choo Tags: [secretive]

Minor change needed: last line added in the User Guide has to be spaced properly. ('format: count' has to start on a new line)

**Comment by suriruhani** [comment]
This is a great command to add! 
Perhaps you can have this stats shown every time the user interacts with the address book, such as after an 'add' command, you can display: 'The address book now has (x+1) contacts!'. Or after a delete or clear command, we can display: 'The address book now has (x-1) or 0 contacts!'.

**Comment by ToonDragon** [comment]
Good command that allows for extension to further functions in the future.  Can be used for many other more complex features.

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 1]
```diff
@@ -82,6 +82,13 @@ public UniquePersonList(UniquePersonList source) {
     }
 
 
+    /**
+     * Returns the count of the list.
+     */
+    public int count() {
```
Comment (ewaldhew) [inline src/seedu/addressbook/data/person/UniquePersonList.java:88]: Maybe better to follow the style of naming for the other stuff and call this `getCount`?

**Review by ewaldhew — COMMENTED** [review]
Interesting and unique little enhancement, nice! 

Close PR after reading @fterh

**Pull request closed without merging**